src/service/TypeService.ts

```diff
@@ -51,8 +51,8 @@ export class TypeService {
   public static async set(
     auth: any,
     type_id: string | null,
-    attachment_key: string,
     target: string,
+    attachment_key: string,
```

Comment:
If you don't update the HTTP handler (at the bottom of this file) this will break things!